signup-front/src/components/organisms/form-sections/DemarcheSection/DemarcheSectionReadOnly.js

```diff
@@ -1,31 +1,31 @@
 import React, { useContext, useEffect, useState } from 'react';
-import { get, has, isEmpty } from 'lodash';
+import { get, has, isEmpty, find } from 'lodash';
 import { FormContext } from '../../../templates/Form';
 import { ScrollablePanel } from '../../Scrollable';
 import { UserContext } from '../../UserContext';
-import { findModifiedFields } from '../../../../lib';
+import { findModifiedFields, findModifiedScopes } from '../../../../lib';
+import './DemarcheSectionReadOnly.css';
 
-export const DemarcheSectionReadOnly = ({ scrollableId }) => {
+export const DemarcheSectionReadOnly = ({ scrollableId, availableScopes }) => {
   const { enrollment, demarches } = useContext(FormContext);
+
   const { demarche: selectedDemarcheId } = enrollment;
   const {
     user: { roles },
   } = useContext(UserContext);
 
   const [modifiedFields, setModifiedFields] = useState([]);
+  const [modifiedScopes, setModifiedScopes] = useState([]);
 
   useEffect(() => {
     if (
       demarches[selectedDemarcheId] &&
       demarches[selectedDemarcheId].state &&
       enrollment
     ) {
-      setModifiedFields(
-        findModifiedFields(
-          get(demarches, selectedDemarcheId, {}).state,
-          enrollment
-        )
-      );
+      const demarcheState = get(demarches, selectedDemarcheId, {}).state;
```

Comment:
use this to avoid adding 500 lines in this PR:

```suggestion
      const demarcheState = merge(
        {},
        get(demarches, 'default', {}).state
        get(demarches, selectedDemarcheId, {}).state
      );
```